common/shell/shell_platform.h

```diff
@@ -0,0 +1,80 @@
+#ifndef SHELL_PLATFORM_H
+#define SHELL_PLATFORM_H
+
+/* Declare Command Enable */
+#define COMMAND_GPIO 1
+#define COMMAND_SENSOR 0
+#define COMMAND_PORT 0
+
+/* Declare Common */
+#define sensor_name_to_num(x) #x,
+#define GET_BIT_VAL(val, n) ((val & BIT(n)) >> (n))
+
+/* Declare GPIO */
+#if COMMAND_GPIO == 1
+#define PINMASK_RESERVE_CHECK 1
+#define GPIO_DEVICE_PREFIX "GPIO0_"
+#define GPIO_RESERVE_PREFIX "Reserve"
+#define NUM_OF_GROUP 6
+#define NUM_OF_GPIO_IS_DEFINE 167
+#define REG_GPIO_BASE 0x7e780000
+#define REG_SCU 0x7E6E2000
+
```

Comment:
Missing header includes for uint32_t as well as gpio_flags_t.

#include <stdint.h>
#include <drivers/gpio.h>